redisless/src/server.rs

```diff
@@ -224,6 +229,23 @@ fn run_command_and_get_response<T: Storage>(
                 storage.write(k.as_slice(), v.as_slice());
                 response
             }
+            Command::MGet(keys) => {
+                // Draft, slow ?
+                // better to add a response formatter module?
+                let mut storage = lock_then_release(storage);
+                let mut final_response = format!("*{}\r\n", keys.len());
+
+                for key in keys {
+                    let response_line = match storage.read(key.as_slice()) {
+                        Some(value) => {
+                            format!("+{}\r\n", std::str::from_utf8(value).unwrap())
+                        }
+                        None => "$-1\r\n".to_string(),
+                    };
+                    final_response.push_str(response_line.as_str());
+                }
+                final_response.as_bytes().to_vec()
+            }
```

Comment:
ways to make this faster?

Reply:
From the [MGET doc](https://redis.io/commands/mget) time complexity is O(N). So it is conform even if we can make it faster at the cost of bigger space complexity.